components/waitwhile/actions/create-customer/create-customer.ts

```diff
@@ -0,0 +1,104 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+import { formatArrayStrings, formatString } from "../../common/utils";
+import constants from "../../common/constants";
+
+export default defineAction({
+  name: "Create Customer",
+  version: "0.0.36",
```

Comment:
Should be version 0.0.1.
```suggestion
  version: "0.0.1",
```